pkg/virt-config/feature-gates.go

```diff
@@ -200,3 +200,15 @@ func (config *ClusterConfig) PanicDevicesEnabled() bool {
 func (config *ClusterConfig) PasstIPStackMigrationEnabled() bool {
 	return config.isFeatureGateEnabled(featuregate.PasstIPStackMigration)
 }
+
+func (config *ClusterConfig) DecentralizedLiveMigrationEnabled() bool {
```

Comment:
https://github.com/kubevirt/kubevirt/pull/15023 This needs to be dropped

Reply:
dropped